cmd/gateway/main.go

```diff
@@ -32,6 +35,7 @@ import (
 type GatewayFlags struct {
 	Identity          identity.Config
 	GenerateTestCerts bool `default:"false" help:"generate sample TLS certs for Minio GW" setup:"true"`
+	Interactive       bool `help:"enable|disable interactive mode" default:"true" setup:"true"`
```

Comment:
With `uplink` we decide to add `non-interactive` flag. It's a little bit weird when to disable bool flag you need `--interactive-false`.
https://github.com/storj/storj/blob/b0f67820f683bedd8a98a0d4a4bac09ef96a4c91/cmd/uplink/cmd/root.go#L26